### PR TITLE
Cherry-pick #10927 to 7.0: Migrate metricbeat docker module to ECS

### LINKED ISSUE #10900: Metricbeat docker module is not ECS ready
`add_docker_metadata` has been migrated to ECS (#9412) and docker autodiscover is going to be migrated too (#10757). For this, all `docker.container.*` fields except `docker.container.labels` are aliases now to `container.*` fields. We should migrate also the metricbeat docker module so it doesn't try to write to the old fields.

### PR DESCRIPTION
**Pull request by jsoriano**
Cherry-pick of PR #10927 to 7.0 branch. Original message: 

Migrate docker metricsets with conflicting fields detected by #10921 to ECS.

Fix #10900

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -20,6 +20,43 @@ https://github.com/elastic/beats/compare/v7.0.0-beta1...master[Check the HEAD di
...
+- Adjust mongodb.status metricset to to ECS. {pull}10368[10368]
+- Refactor munin module to collect an event per plugin and to have more strict field mappings. `namespace` option has been removed, and will be replaced by `service.name`. {pull}10322[10322]
+- Change the following fields from type text to keyword: {pull}10318[10318]
+  - ceph.osd_df.name
+  - ceph.osd_tree.name
+  - ceph.osd_tree.children
+  - kafka.consumergroup.meta
+  - kibana.stats.name
+  - mongodb.metrics.replication.executor.network_interface
+  - php_fpm.process.request_uri
```
Comment (kaiyan-sheng) [inline CHANGELOG.next.asciidoc]: oh no, too many lines here!

Comment (jsoriano) [reply CHANGELOG.next.asciidoc]: You were fast! :smile: Fixed.

File: CHANGELOG.next.asciidoc [hunk 2]
```diff
@@ -20,6 +20,10 @@ https://github.com/elastic/beats/compare/v7.0.0-beta1...master[Check the HEAD di
 
 *Metricbeat*
 
+- Migrate docker module to ECS. {pull}10927[10927]
+
+*Packetbeat*
```
Comment (jsoriano) [inline CHANGELOG.next.asciidoc]: This entry was actually missing here :thinking:

Comment (kaiyan-sheng) [reply CHANGELOG.next.asciidoc]: hmm what happened with cherry-pick script hmm

**Pull request merged**